test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
```

Comment:
it's not a perf test, the QPS was client-side and the throttling was causing flakes in failing the proxyHTTPCallTimeout check